user-guide/Reference/Skyline_DataMiner_Folder/More_information_on_certain_files_and_folders/MaintenanceSettings_xml.md

```diff
@@ -214,6 +214,23 @@ The following types of alarm events will not be combined in a consolidated alarm
 > - [Alarm linking](xref:Alarm_linking)
 > - [Declutter your alarm tree](https://community.dataminer.services/declutter-your-alarm-tree/ "Alarm squashing")
 
+### AlarmSettings.PersistParameterLatchState
+
+From DataMiner 10.4.9/10.5.0 onwards<!--RN 39495-->, parameter latch states are no longer persistent by default. This means they will reset after every DataMiner restart.
```

Comment:
To make this more future-proof, I'd prefer to avoid using "no longer" (especially since at some point the version info will get removed here).
```suggestion
From DataMiner 10.4.9/10.5.0 onwards<!--RN 39495-->, parameter latch states are not persistent by default. This means they will reset after every DataMiner restart.
```